ansible_base/authentication/authenticator_plugins/local.py

```diff
@@ -37,7 +38,21 @@ def authenticate(self, request, username=None, password=None, **kwargs):
         # This auth class doesn't create any new local users, so we just need to make sure
         # it has an AuthenticatorUser associated with it.
         if user:
-            AuthenticatorUser.objects.get_or_create(uid=username, user=user, provider=self.database_instance)
+            logger.info(f"user 2: {user.__dict__}")
```

Comment:
If this is debug log make sure it gets removed, if not make it more descriptive.